### PR TITLE
Updated default ModuleVersion in ModuleManifest

### PR DESCRIPTION
**Pull request by LDSpits**
This pull request updates the New-ModuleManifest command to generate a ModuleVersion manifest property with 0.0.1 instead of 1.0. specified by the discussion in #4823.

### CODE REVIEW

File: src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs [hunk 1]
```diff
@@ -940,12 +940,12 @@ protected override void EndProcessing()
...
+                    filePath,
 #if UNIX
-                    new UTF8Encoding(false), // UTF-8, no BOM
-#else
-                    EncodingConversion.Unicode, // UTF-16 with BOM
-#endif
+                    new UTF8Encoding(false), // UTF-8, no BOM
+#else
+                    EncodingConversion.Unicode, // UTF-16 with BOM
+#endif
```
Comment (daxian-dbw) [inline src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs]: Is this part of change caused by some character encoding? If no intentional change was made, you can get rid of it by using `git reset HEAD^` and then `git add -p` (ignore this change section when adding).

**Review by daxian-dbw — CHANGES_REQUESTED** [review]

File: src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs (hunk 1 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs]: As part of https://github.com/PowerShell/PowerShell/pull/4761#issuecomment-327852126, this file has mixed EOL characters.  We can accept it to make the EOLs the same for this file or ignore it as I'm fixing this across the entire codebase in a separate PR.

Comment (daxian-dbw) [reply src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs]: OK, then we can accept this part of the changes.

**Review by daxian-dbw — APPROVED** [review]

**Review by adityapatwardhan — APPROVED** [review]
LGTM

**Pull request merged**